File: Strings/pagina05/2356/2356.cpp
#include <iostream>
#include <string>
using namespace std;

int main(){

    string becteria_dna, code_genetico;

    while(getline(cin, becteria_dna) and getline(cin, code_genetico)){

        if(becteria_dna.find(code_genetico) != string::npos){
            cout << "Resistente\n";
        } else {
            cout << "Nao resistente\n";
        }
    }

    return 0;
}

File: Strings/pagina05/2356/2356.php
<?php 

    while(($becteria_dna = readline()) !== false && ($code_genetico = readline()) !== false){

        $becteria_dna = trim($becteria_dna);
        $code_genetico = trim($code_genetico);

        if(strpos($becteria_dna, $code_genetico) !== false){
            echo "Resistente\n";
        } else {
            echo "Nao resistente\n";
        }
    }
?>